src/consensus/condition_costs.py

```diff
@@ -3,7 +3,7 @@
 
 class ConditionCost(Enum):
     # Condition Costs
-    AGG_SIG = 20
+    AGG_SIG = 92  # 1 ms BLS verify = 10,000 clvm cost / 108 cost multiplier
```

Comment:
we don't enforce a max cost yet though, right?

Reply:
We do in testnet